backend/utils.py

```diff
@@ -1765,4 +1765,73 @@ def add_or_update_organization_url(organization_id, url, scraping_result=None, a
         
     except Exception as e:
         logging.error(f"[add_or_update_organization_url] add_or_update_organization_url: something went wrong. {str(e)}")
-        raise
\ No newline at end of file
+        raise
+    
+def create_organization_usage(organization_id, period_start, period_end):
+    if not organization_id:
+        return {"error": "Organization ID not found."}
+
+    logging.info(
+        f"[create_organization_usage] Processing usage record for organization: {organization_id}"
+    )
+
+    credential = DefaultAzureCredential()
+    db_client = CosmosClient(AZURE_DB_URI, credential, consistency_level="Session")
+    db = db_client.get_database_client(database=AZURE_DB_NAME)
+    container = db.get_container_client("organizationUsage")
+
+    try:
+        # Check if usage record already exists
+        query = "SELECT * FROM c WHERE c.organization_id = @organization_id"
+        parameters = [{"name": "@organization_id", "value": organization_id}]
+        results = list(
+            container.query_items(
+                query=query, parameters=parameters, enable_cross_partition_query=True
+            )
+        )
+
+        if results:
+            usage_item = results[0]
+            usage_item["periodStart"] = period_start
+            usage_item["periodEnd"] = period_end
+            # If updating the subscription/period, we might want to reset usage or keep it?
+            # The request says "include ... a current usage with a 0 value".
+            # We will assume updating implies starting/refreshing the period usage.
+            usage_item["current_usage"] = 0 
+            usage_item["updatedAt"] = int(datetime.now(timezone.utc).timestamp())
+            
+            container.replace_item(item=usage_item["id"], body=usage_item)
```

Comment:
@Alexjes98 updating the subscription/period, we might want to reset usage or keep it?